src/uu/install/src/install.rs

```diff
@@ -98,6 +98,9 @@ enum InstallError {
 
     #[error("failed to access {}: Not a directory", .0.quote())]
     NotADirectory(PathBuf),
+
+    #[error("cannot overwrite directory {} with non-directory", .0.quote())]
+    OverrideDirectoryFailed(PathBuf),
```

Comment:
Here you might want to add a second `PathBuf` because GNU `install` shows both `to` and `from` in its error message:
```
$ install file dir1/
install: cannot overwrite directory 'dir1/file' with non-directory 'file'
```